ocw_data_parser/utils.py

```diff
@@ -280,3 +286,43 @@ def parse_all(  # pylint: disable=too-many-arguments, too-many-locals
             parser.export_parsed_json(
                 s3_links=s3_links, upload_parsed_json=perform_upload
             )
+
+
+def convert_to_vtt(loaded_json):
+    """
+    Convert a json file with srt captions data to vtt format
+
+    Args:
+        loaded_json (dict): The content of JSON file
+
+    Returns:
+        dict: copy of the json file with the _datafield_file converted to vtt format
+    """
+    if loaded_json["id"].endswith(".vtt"):
+        return None
+    new_json = dict(loaded_json)
+
+    new_json["id"] = re.sub(r".srt$", ".vtt", loaded_json["id"])
```

Comment:
Looking at the output it may be good to also update the filenames in `unique_identifier`, `_id`, `_path`

Reply:
I don't see those fields getting used in the json file file processing, and they don't seem to be present in the output. Are they being used somewhere?